igvm/vm.py

```diff
@@ -430,7 +430,7 @@ def info(self):
             result['status'] = 'new'
         return result
 
-    def build(self, run_puppet=True, debug_puppet=False, postboot=None):
+    def build(self, run_puppet=True, debug_puppet=False, postboot=None, rebuild=False):
```

Comment:
Why do you call this `rebuild` when it only controls clearing of Puppet certificate?

Reply:
Because it is only required in the rebuild case.

Reply:
Flags' names should tell us what they do, not why we call them.

Reply:
Renamed the flag to 'clean_cert'